<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clocking</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,5">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0" columnstretch="0,0,1,0" rowminimumheight="0,0,0,0,50">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="3" column="1" colspan="3">
       <widget class="QTableWidget" name="clockTableWidget">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0,0,1,0,1,0,0,1">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="4">
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="globalRadioButton">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Global</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="userLabel">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>User:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="categoryLabel">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string>Category:</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QLabel" name="totalIncomeLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Total Income:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="totalTimeLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Total Time:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QPushButton" name="clockButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Clock In</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0,0,0,1,0,0">
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="1" column="1">
       <widget class="QComboBox" name="userBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>User</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="categoryBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="addCategoryButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add Category</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="addUserButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add User</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_Invoice"/>
    <addaction name="actionExport_All_Invoices"/>
    <addaction name="actionSet_Default_Invoice_Path"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuButtons">
    <property name="title">
     <string>Buttons</string>
    </property>
    <addaction name="actionAdd_Button"/>
    <addaction name="actionAssign_Buttons"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="userAddAction_2"/>
    <addaction name="actionEdit_User"/>
    <addaction name="actionDelete_User"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Category_2"/>
    <addaction name="actionEdit_Category"/>
    <addaction name="actionDelete_Category"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Company"/>
    <addaction name="separator"/>
    <addaction name="actionClock"/>
   </widget>
   <widget class="QMenu" name="menuEmailing">
    <property name="title">
     <string>Emailing</string>
    </property>
    <addaction name="actionEMail_Now"/>
    <addaction name="separator"/>
    <addaction name="actionSetup_Emailing"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Email_Template"/>
    <addaction name="actionAssign_Days_to_Send_Emails"/>
    <addaction name="actionSet_Users_Invoices_to_Email"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuEmailing"/>
   <addaction name="menuButtons"/>
  </widget>
  <action name="actionAdd_Button">
   <property name="text">
    <string>Add Button</string>
   </property>
  </action>
  <action name="actionExport_Invoice">
   <property name="text">
    <string>Export Invoice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdd_User">
   <property name="text">
    <string>Add User</string>
   </property>
  </action>
  <action name="actionAdd_Category">
   <property name="text">
    <string>Add Category</string>
   </property>
  </action>
  <action name="categoryAddAction">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="categoryDeleteAction">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="categoryEditAction">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="userAddAction">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="userEditAction">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="userDeleteAction">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="userAddAction_2">
   <property name="text">
    <string>Add User</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEdit_User">
   <property name="text">
    <string>Edit User</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDelete_User">
   <property name="text">
    <string>Delete User</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdd_Category_2">
   <property name="text">
    <string>Add Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+A</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEdit_Category">
   <property name="text">
    <string>Edit Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDelete_Category">
   <property name="text">
    <string>Delete Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+D</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExport_All_Invoices">
   <property name="text">
    <string>Export All Invoices</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClock">
   <property name="text">
    <string>Clock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAssign_Buttons">
   <property name="text">
    <string>Assign Buttons</string>
   </property>
  </action>
  <action name="actionSet_Email_Template">
   <property name="text">
    <string>Set Email Template</string>
   </property>
  </action>
  <action name="actionAssign_Days_to_Send_Emails">
   <property name="text">
    <string>Assign Days to Send Emails</string>
   </property>
  </action>
  <action name="actionSet_Default_Invoice_Path">
   <property name="text">
    <string>Set Default Invoice Path</string>
   </property>
  </action>
  <action name="actionSetup_Emailing">
   <property name="text">
    <string>Set Emailing</string>
   </property>
  </action>
  <action name="actionSet_Users_Invoices_to_Email">
   <property name="text">
    <string>Set Users Invoices to Email</string>
   </property>
  </action>
  <action name="actionEMail_Now">
   <property name="text">
    <string>Email Now</string>
   </property>
  </action>
  <action name="actionSet_Company">
   <property name="text">
    <string>Set Company</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
